apps/site/assets/js/event-page-setup.js

```diff
@@ -29,9 +31,56 @@ function addInternetExplorerPolyfills() {
   stickyPolyfill();
 }
 
+function setupEventPopups() {
+  /**
+   * Why not use Bootstrap's popover? It's not equipped to handle having lots of
+   * interactive content, doesn't support a close button, and as implemented the
+   * UX for assistive tech users probably isn't ideal. For more, see "Making
+   * popovers work for keyboard and assistive technology users" in
+   * https://getbootstrap.com/docs/4.6/components/popovers/
+   *
+   * This implementation uses ally-dialog (https://a11y-dialog.netlify.app) as a
+   * base to create accessible dialog windows for various use cases.
+   */
+  const calendarView = document.querySelector(".m-events-hub--list-view");
+
+  // find all the event popup HTML elements
+  const eventPopups = calendarView.querySelectorAll(".m-event-overlay");
+  for (let i = 0; i < eventPopups.length; i++) {
+    const el = eventPopups[i];
+    const dialog = new A11yDialog(el);
+
+    // Hide popup when user clicks outside popup, or on a different event
+    document.addEventListener("click", e => {
+      const path = e.composedPath();
+      const eventClicked = path.find(
+        x => x.className === "m-event-calendar__event"
```

Comment:
I don't remember: do these files get compiled down so they use an earlier version of JS? Otherwise I'd suspect that older browsers don't support this syntax.

Reply:
Erm... I'd have to ask my teammates, but I see there is precedent for using the updated syntax in other JS files